### ISSUE TITLE
detokenization problem: `\n` removed after `\t` from `wsafter`

### ISSUE THREAD

**Issue by sassbalint** [dlt-rilmta/quntoken#39]
`input.txt` file, where `TAB` means a TAB:

```txt
egyik
TAB

másik
```

Command:
```bash
cat input.txt | docker run --rm -i mtaril/emtsv tok
```

Output:
```txt
form	wsafter
egyik	"\n\t"

másik	"\n"
```
I think, this is problematic from detokenization point of view because in this case detokenization yields:
```txt 
egyik
TABmásik
```
which is not identical with the original input.

So correct output should be something like:
```txt
form	wsafter
egyik	"\n\t\n\n"

másik	"\n"
```

_Workaround:_ delete TABs from end of lines before calling `quntoken`.

**Comment by dlazesz** [comment]
Which quntoken version do you use?

The latest version yields the following for me:

```
form	wsafter
egyik	"\n\t\n\n"

másik	"\n"
```

This is the expected output.

**Comment by sassbalint** [comment]
I just use `docker pull mtaril/emtsv` and run by `docker run --rm -i mtaril/emtsv tok`.

**Comment by dlazesz** [comment]
That is outdated.

[This patch](https://github.com/dlt-rilmta/quntoken/pull/38) has been merged into Quntoken at [3.1.7](https://github.com/dlt-rilmta/quntoken/releases/tag/v3.1.7), and [Quntoken is updated in emtsv](https://github.com/dlt-rilmta/emtsv/commit/9422a4794cc2b8f64f2a830a970f8c968ff229cc#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552L13).
emtsv is just not yet released to include this fix, so spin your own docker image or use newest quntoken separately.

**Comment by sassbalint** [comment]
Thanks @dlazesz Balázs! :) Can be closed.
